api/backends/index.js

```diff
@@ -2,15 +2,13 @@ import axios from 'axios'
 
 /* eslint-disable */
 export let visualisationApiUrl = process.env.visualisationApiUrl
-export let loginUrl = process.env.loginUrl
 export let dssauthUrl = process.env.dssauthUrl
 export let discoveryApiUrl = process.env.discoveryApiUrl
 export let environment = process.env.environment
 
 const appMode = process.env.NODE_ENV
 if (appMode !== 'production') {
-  visualisationApiUrl = "https://dafni-nivs-nivs-api-review-dev-hkw12f.staging.dafni.rl.ac.uk"
-  loginUrl = "https://dafni-dss-login-app-jwt-review-dev-h79t6w.staging.dafni.rl.ac.uk"
```

Comment:
I guess there's no replacement for this because the redirect will be handled by the ingress?

Reply:
Its taken care of by the keycloak package not the ingress